docs/http-body.md

```diff
@@ -0,0 +1,53 @@
+# Body
+
+## Standard properties
+
+Error responses from publiq APIs always include at least the 4 following properties defined by the [RFC7807](https://datatracker.ietf.org/doc/html/rfc7807) standard:
+
+*   `type` - A URL used to identify the problem. You can use this to differentiate between errors inside your application. Make sure to always compare the full URL and not just the last part, since some APIs might share similar yet different error types.
+*   `title` - A short, human-readable summary of the problem type.
+*   `detail` - A human-readable summary about this specific instance of the problem.
+*   `status` - The HTTP status code generated by the origin server for this occurrence of the problem.
+
+For example:
+
+```json
+{
+  "type": "https://api.publiq.be/probs/uitpas/invalid-uitpasnumber",
+  "title": "UiTPAS number invalid",
+  "detail": "UiTPAS numbers must be exactly 13 digits.",
+  "status": "400"
+}
+```
+
+## Extensions
+
+Aside from the standardized properties above, the RFC7807 spec also allows for custom properties. The following extra properties are commonly used on publiq APIs to provide more information in specific situations.
```

Comment:
echt heel common is dat nu ook niet momenteel. maar 1 uitpas endpoint waar ik dat doe

Reply:
Nee maar ik zou dat bv ook op UDB op sommige plaatsen gebruiken. Met "common" bedoel ik eerder dat het op meerdere APIs hetzelfde gebruikt wordt

Reply:
Ik heb het aangepast naar `sometimes` ipv `commonly`